### PR TITLE
Update Community Building Committee charter

### PR DESCRIPTION
**Pull request by jasongrout**
### Background or context to help others understand the change.

The guidelines for organizations have changed with the move to the EC/SSC governance, so we are updating the charter to reflect the new name of this existing body.

Also, this is a chance to update the charter to reflect how our responsibilities are evolving since the formation of the committee.

### A brief summary of the change.

This reflects a name change (Community Working Group) to follow current guidelines about Jupyter organizations, and also updates some aspects of the charter to reflect current responsibilities.

### What is the reason for this change?

Since the governance has introduced new terminology, we are updating our charter.

### Alternatives to making this change and other considerations.

> ## The process ❗
> 
> The process for changing the governance pages is as follows:
> 
> * Open a pull request **in draft state**. This triggers a discussion and iteration phase
>   for your proposed changes.
> * When you believe enough discussion has happened,
>   **move the pull request to an active state**. This triggers a vote.
> * During the voting phase, no substantive changes may be made to the pull request.
> * The Executive Council and Software Steering Council will vote, and at the end of voting the pull request is merged or closed.
> 
> The discussion phase is meant to gather input and multiple perspectives from the community.
> Make sure that the community has had an opportunity to weigh in on
> the change **before calling a vote**. A good rule of thumb is to ask several Council
> members if they believe that it is time for a vote, and to let at least one person review
> the pull request for structural quality and typos.

## Votes

Since this is a working group (under sole purview of the EC), the EC is the voting body for this PR.

> The decision will be determined by a simple majority of non-blank votes for binary decisions (i.e., approving a proposal) and ranked choice for multi-class decisions (one among many, or several among many). The sponsor may update the proposal at any point during the voting period, in which case the voting period will be reset.

**The voting period for the PR is from 25 Jul 2023 to 25 Aug 2023 anywhere on 🌍**

@afshin 
- [x] Yes
- [ ] No
- [ ] Abstain

@Ruv7
- [ ] Yes
- [ ] No
- [ ] Abstain

[@ellisonbg](https://github.com/ellisonbg)
- [x] Yes
- [ ] No
- [ ] Abstain

[@fperez](https://github.com/fperez)
- [x] Yes
- [ ] No
- [ ] Abstain

[@jasongrout](https://github.com/jasongrout)
- [ ] Yes
- [ ] No
- [x] Abstain

[@blink1073](https://github.com/blink1073)
- [ ] Yes
- [ ] No
- [ ] Abstain

### CODE REVIEW

**Comment by jasongrout** [comment]
CC @jupyter/community-working-group

**Comment by jasongrout** [comment]
Please note that this proposal removes the definitions of DEI in our charter. I feel that it would be more appropriate for the DEI committee to define these terms.

**Review by Ruv7 — APPROVED** [review]

File: communitybuildingcommittee.md
Comment (Ruv7) [inline communitybuildingcommittee.md:1]: I want to say in our last meeting we agreed to the following: 

- Jupyter Community Building Working Group as the full name
- JCB as the acronym

But there was quite a bit of back and forth. Please confirm. Note we'll need to update this in the governance documentation and our google group. Putting this note here so we don't forget.

**Review by marthacryan — APPROVED** [review]

**Comment by jasongrout** [comment]
I've made the change to Jupyter Community Building working group (JCB). I think the final step is EC approval, which I can bring up in the EC meeting tomorrow.

File: communitybuildingworkinggroup.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Community Building Working Group
+
+The Jupyter Community Building Working Group (JCB) acts on behalf of the Project Jupyter
+Executive Council with the objective of growing, building, and connecting the
+Jupyter community of users and contributors. A targeted outcome of this
```
Comment (ellisonbg) [inline communitybuildingworkinggroup.md:5]: I would maybe say that this WG works with the DEI standing committee to improve diversity, equity, and inclusion in our community.

File: communitybuildingworkinggroup.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+- Collaborate with the Jupyter Diversity, Equity, and Inclusion (DEI)
+  standing committee to find new ways to make a positive impact in
+  Project Jupyter.
+- Partner with the DEI standing committee to train and mentor community event
+  leadership in DEI efforts for their events.
+- Help the DEI standing committee measure impact of DEI efforts in Jupyter events where feasible.
+
+## Reporting
+
+This working group will report to the Executive Council quarterly on all areas of
```
Comment (ellisonbg) [inline communitybuildingworkinggroup.md:67]: We should have a reporting policy that applies to all Working Groups and Standing Committees in a uniform manner. I would rather keep this detail out of the individual charters (but am fine with the quarterly reporting cadence.)

File: communitybuildingworkinggroup.md (hunk 1 above)
Comment (ellisonbg) [reply communitybuildingworkinggroup.md:5]: We say that below in the more detailed section, so I would repeat that here in summary form.

**Review by fperez — APPROVED** [review]

File: communitybuildingworkinggroup.md (hunk 2 above)
Comment (jasongrout) [reply communitybuildingworkinggroup.md:67]: Agreed - for a future governance update.

**Pull request merged**